asmcomp/amd64/emit.mlp

```diff
@@ -171,7 +171,18 @@ let emit_label lbl =
 
 let label s = sym (emit_label s)
 
-let def_label s = D.label (emit_label s)
+module Int = Numbers.Int
+
+(* For Spacetime, keep track of code labels that have been emitted.  *)
+let used_labels = ref Int.Set.empty
+
+let mark_used lbl =
+  if Config.spacetime && not (Int.Set.mem lbl !used_labels) then
```

Comment:
I'd prefer to see `begin` and `end` around the "`then`" clause.

Reply:
Fixed.